mysql-test/columnstore/basic/r/mcol-4407.result

```diff
@@ -0,0 +1,32 @@
+DROP DATABASE IF EXISTS mcol_4407;
```

Comment:
Plz move the tests into bugfixes instead.

Reply:
Sure. will fix it.